src/zip/zip.go

```diff
@@ -74,17 +79,43 @@ func (zipHandler ZipHandler) Unzip(zipFilePath string) error {
 
 	// loop over contents
 	for _, f := range zipReader.File {
-		errorList = zipHandler.ExtractAndUploadSingleFile(f, zipPassword, zipFilePath, errorList)
+		errorList = zipHandler.ExtractAndUploadSingleFile(f, zipPassword, zipFileName, errorList)
+	}
+
+	// if errorList has contents -> move zip file from unzip -> unzip/failure
+	if len(errorList) > 0 {
+		slog.Info("Error list length over zero")
+		zipHandler.MoveZip(blobPath, utils.FailureFolder)
+	} else {
+		// else -> move zip file from unzip -> unzip/success
+		slog.Info("Error list length is zero")
+		zipHandler.MoveZip(blobPath, utils.SuccessFolder)
 	}
+
+
 	// Upload error info if any
-	err = zipHandler.UploadErrorList(zipFilePath, errorList, err)
+	err = zipHandler.UploadErrorList(blobPath, errorList, err)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
+// MoveZip moves a file from 'unzip' into the specified subfolder e.g. 'success', 'failure'
+func (zipHandler ZipHandler) MoveZip(blobPath string, subfolder string) {
+	slog.Info("About to move file", slog.String("blobPath", blobPath), slog.String("destination subfolder", subfolder))
+	// url must include the container name e.g. 'sftp/unzip/cheeseburger.zip' vs 'unzip/cheeseburger.zip'
```

Comment:
it might be helpful to denote somehow that the second example is a 'path' and doesn't need the container name

Reply:
I've updated this comment but am welcomed to better suggestions/rewriting